ChatApplication/Models/Message.cs

```diff
@@ -2,23 +2,19 @@
 using System.Runtime.Serialization;
 using System.Text.Json.Serialization;
 
-
 namespace ChatApplication.Models
 {
     public class Message
     {
         public int Id { get; set; }
-        public string UserName { get; set; }
         public string Text { get; set; }
         public DateTime When { get; set; }
-
         public string UserId { get; set; }
-
         public virtual AppUser Sender { get; set; }
 
         public Message()
         {
-            When = DateTime.UtcNow;
+            When = DateTime.Now;
```

Comment:
what is the difference between  UtcNow and Now why did you change that?

Reply:
Date.Time.Now gives the time and date in real time in your local,but Date.Time.UtcNow calls the Greenwich Mean   time zone